e2e_test/network_test.go

```diff
@@ -0,0 +1,331 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/swaggest/assertjson"
+
+	"github.com/hetznercloud/hcloud-go/v2/hcloud"
+)
+
+func TestNetwork(t *testing.T) {
+	t.Parallel()
+
+	out, err := runCommand(t, "network", "create")
+	assert.Empty(t, out)
+	require.EqualError(t, err, `required flag(s) "ip-range", "name" not set`)
+
+	networkName := withSuffix("test-network")
+	networkID, err := createNetwork(t, networkName, "--ip-range", "10.0.0.0/24")
+	require.NoError(t, err)
+
+	_, err = createNetwork(t, networkName, "--ip-range", "10.0.1.0/24")
+	require.Error(t, err)
+	assert.Regexp(t, `^name is already used \(uniqueness_error, [0-9a-f]+\)$`, err.Error())
```

Comment:
Can we disable this assertion until the API response is fixed? To validate that the CI & Coverage works as expected

Reply:
Can you comment this out instead of checking for the wrong response? Otherwise the tests start failing once this is fixed.